app/src/main/java/com/yeonae/chamelezone/view/place/PlaceDetailActivity.kt

```diff
@@ -75,15 +79,16 @@ class PlaceDetailActivity : AppCompatActivity(), PlaceInfoContract.View {
 
                 Log.d("PlaceDetailActivity nameBar", nameBar.toString())
                 Log.d("PlaceDetailActivity tabBar", tabBar.toString())
+
                 layoutParams = tool_bar.layoutParams.apply {
                     height = nameBar + tabBar
                 }
-
-                app_bar.addOnOffsetChangedListener(OnOffsetChangedListener { appBarLayout, verticalOffset ->
-                    if (abs(verticalOffset)-appBarLayout.totalScrollRange == 0) {
+                app_bar.addOnOffsetChangedListener(AppBarLayout.OnOffsetChangedListener { appBarLayout, verticalOffset ->
+                    if (abs(verticalOffset) - appBarLayout.totalScrollRange > 0) {
                         layout_visibility.visibility = View.VISIBLE
```

Comment:
```suggestion
                        layout_visibility.isVisible = true
```
이런거도 있어요 isInvisible isGone도 있어요
참고만 하세요!

Reply:
참고하겠습니다!